### PR TITLE
Fix/set period when exporting tracewarnings

### PR DESCRIPTION
**Pull request by FelixRottler**
- adds the missing period when exporting trace warnings

### CODE REVIEW

**Review by roesslerj — DISMISSED** [review]

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java [hunk 1]
```diff
@@ -70,7 +68,7 @@ private QRCodePosterTemplateAndroid buildAndroidProtoStructure(
         .setTemplate(qrTemplateLoader.loadAndroidTemplateAsBytes())
         .setQrCodeSideLength(templateConfig.getQrCodeSideLength())
         .setDescriptionTextBox(QRCodeTextBoxAndroid.newBuilder()
-            .setOffsetX(textBoxConfig.getOffsetX().floatValue()).setOffsetY(textBoxConfig.getOffsetY().floatValue())
+            .setOffsetX(textBoxConfig.getOffsetX().intValue()).setOffsetY(textBoxConfig.getOffsetY().intValue())
```
Comment (pithumke) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java:71]: Didn't Android want to have the float? (see [here](https://github.com/corona-warn-app/cwa-server/pull/1314/files#diff-45fb40b80c6e2f60b43366005db12e9bf4ff8c288645ad642988316e990e7ab5R68))

Comment (FelixRottler) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java:71]: I got a compile error

**Review by pithumke — DISMISSED** [review]

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java (hunk 1 above)
Comment (pithumke) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java:71]: That value is definitely a float (see [here](https://github.com/corona-warn-app/cwa-server/blob/release/2.0/common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/pt/qr_code_poster_template.proto#L18)). Maybe something broke during merge?

Comment (ioangut) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java:71]:
```suggestion
            .setOffsetX(textBoxConfig.getOffsetX().floatValue()).setOffsetY(textBoxConfig.getOffsetY().floatValue())
```

**Review by pithumke — APPROVED** [review]

**Pull request merged**